ietf/static_src/css/iab.scss

```diff
@@ -45,18 +45,14 @@
     background-color: var(--color-iab-blue-2) !important;
 }
 
-.iab-bg-navy {
+.iab-bg-blue-3 {
```

Comment:
My only nagging feeling is that this `iab.scss` file is doing too much. It's got multiple new styles, some modifiers of existing styles, and utility classes (e.g. `.iab-bg-blue-3`).

Gicen that the file is only 183 lines and exists solely to add/override stuff specifically for the IAB site, this is probably manageable now. But if this will grow it might make sense to break it up a bit in the future.

Reply:
yep, that makes perfect sense to me. We might e.g. make an `IAB` directory under `css` where we can break them up and group them logically.